packages/render-fidelity-tools/src/components/renderers/babylon-viewer.ts

```diff
@@ -132,11 +133,21 @@ export class BabylonViewer extends LitElement {
 
     this[$scene].stopAllAnimations();
 
-    // load and prefilter HDR texture (the size of cubmap is set to be 256 for
-    // all renderers)
-    const environment = new HDRCubeTexture(
-        scenario.lighting, this[$scene], 256, false, false, false, true);
+    // For scenarios using these two hdr files, real time filters looks better
+    // than prefilter. Also, when enable real time filter, prefilter should not
+    // be enabled at the same time.
+    const needRealTimeFilter =
+        (lightingBaseName == 'spruit_sunrise_1k_HDR' ||
+         lightingBaseName == 'spot1Lux');
+
+    // the size of cubemap is 256 for all other renderers
+    const environment = needRealTimeFilter ?
+        new HDRCubeTexture(
+            scenario.lighting, this[$scene], 256, false, false, false, false) :
+        new HDRCubeTexture(
+            scenario.lighting, this[$scene], 256, false, false, false, true);
```

Comment:
I think you could further simplify this by skipping the ternary and putting the `needRealTimeFilter` boolean directly into the last input.

Reply:
Good idea!